sdk/communication/AzureCommunicationChat/Source/AzureCommunicationChatClient.swift

```diff
@@ -16,19 +16,24 @@ import Foundation
 // swiftlint:disable function_body_length
 // swiftlint:disable type_body_length
 
+extension CharacterSet {
+    static let azureUrlQueryAllowed = urlQueryAllowed.subtracting(.init(charactersIn: "!*'();:@&=+$,/?"))
+    static let azureUrlPathAllowed = urlPathAllowed.subtracting(.init(charactersIn: "!*'()@&=+$,/:"))
+}
+
 public final class AzureCommunicationChatClient: PipelineClient, PageableClient {
     public func continuationUrl(forRequestUrl _: URL, withContinuationToken token: String) -> URL? {
         return URL(string: token)
     }
 
     /// API version of the  to invoke. Defaults to the latest.
     public enum ApiVersion: String {
-        /// API version "2020-09-21-preview2"
-        case v20200921preview2 = "2020-09-21-preview2"
+        /// API version ""
+        case v = ""
 
         /// The most recent API version of the
         public static var latest: ApiVersion {
-            return .v20200921preview2
+            return .v
         }
     }
```

Comment:
@sacheu looks like there's still an issue here with how Autorest.Swift is (or rather, is not) generating the api version field.

Reply:
I have manually reverted this and updated the issue in the Autorest.Swift repo to fix this.